test/unit/api/request.spec.js

```diff
@@ -1,182 +1,294 @@
-const chai = require('chai');
+const chai = require("chai");
 const { expect } = chai;
-const sinon = require('sinon');
-const proxyquire = require('proxyquire');
-const { Errors, Document } = require('../../../');
-const { NotImplementedError } = Errors;
+const sinon = require("sinon");
+const proxyquire = require("proxyquire");
+const { Document } = require("../../../");
 
-chai.use(require('sinon-chai'));
+chai.use(require("sinon-chai"));
```

Comment:
Were double quotes in the eslint, I won't object. Just curious if that's what happened here. We may prefer single long-term.

Reply:
Not a blocker. But we should change that to be single in the future.

Reply:
Aha, that just came up when I hit the lint-fix command

Reply:
> Not a blocker. But we should change that to be single in the future.

Got it 😄 👍